assets/src/localStorage.ts

```diff
@@ -0,0 +1,34 @@
+import { RouteId } from "./skate"
+
+const APP_STATE_KEY = "mbta-skate-state"
+
+const SAVED_KEYS = ["selectedRouteIds"]
+
+interface PersistedState {
+  selectedRouteIds: RouteId[]
+}
+
+export const saveState = (state: PersistedState) => {
+  const perstableState = filter(state, SAVED_KEYS)
+  const serializedState = JSON.stringify(perstableState)
+  localStorage.setItem(APP_STATE_KEY, serializedState)
+}
+
+export const loadState = (): PersistedState | undefined => {
+  const serializedState = localStorage.getItem(APP_STATE_KEY)
```

Comment:
Is there any namespace collision between the name of this file and this reference to `localStorage`?

Reply:
No, names of files don't affect anything in JS.